pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1821,6 +1837,11 @@ func configToXencfg(config types.DomainConfig, status types.DomainStatus,
 		if ds.Format == zconfig.Format_CONTAINER {
 			continue
 		}
+		err := checkDiskFormat(ds)
+		if err != nil {
+			log.Errorf("%v", err)
```

Comment:
FWIW you can just go a log.Error(err) here.

Reply:
For me, it is failing with the following error:
# github.com/lf-edge/eve/pkg/pillar/cmd/domainmgr
cmd/domainmgr/domainmgr.go:1842:14: cannot use err (type error) as type string in argument to logrus.Errorf

Reply:
log.Error(err) not log.Errorf
We use that pattern in lots of places.

Reply:
Oh, my bad. I will update it.